### ISSUE TITLE
Problem importing manifest.json - Version 0.4.1

### ISSUE THREAD

**Issue by igor-ribeiro** [jeffposnick/create-react-pwa#3]
The version of `react-scripts` I'm using is 0.4.1 and adding

``` html
<link rel="manifest" href="manifest.json">
```

does not work because there is a loader on `webpack.config.prod.js` to resolve links with href. The output is:

``` html
<link rel="manifest" href="[object" object]>
```

To fix that I edited the `/node_modules/react-scripts/config/webpack.config.prod.js` like that:

``` javascript
// JSON is not enabled by default in Webpack but both Node and Browserify
// allow it implicitly so we also enable it.
{
  test: /\.json$/,
  exclude: /\/manifest.json$/, // ADD THIS!
  loader: 'json'
},
```

And add this block inside the loaders:

``` javascript
// A special case for manifest.json to place it into build root directory.
{
  test: /\/manifest.json$/,
  loader: 'file',
  query: {
    name: 'manifest.json?[hash:8]'
  }
},
```

Then link on `index.html` this way 

``` html
<link rel="manifest" href="./manifest.json">
```

With this configuration you don't need to add `cp manifest.json build/` to the build script on `package.json`

**Comment by jeffposnick** [comment]
Hrm, that's unfortunate.

I don't want to force folks to muck around with the config that ships with `create-react-app`.

I know next to nothing about webpack, but I did find https://github.com/webpack/html-loader#root-relative-urls, which suggests that any URLs that start with `/` will be excluded from translation. Using `<link rel="manifest" href="/manifest.json">` does seem to work, but that implies that you're going to serve `manifest.json` from a top-level path, which might not always be the case.

I feel like there _must_ be some way to escape a non-root `href` value inline in the HTML to tell webpack not to translate it, without having to resort to editing the config and adding it to `exclude`, but I'm at a lost for how that could be done.

CC: @addyosmani because I know he's been playing with webpack lately 😄

**Comment by igor-ribeiro** [comment]
Folks of `create-react-app` will ship this fix on version 0.4.2, maybe an upgrade here later will be fine. Anyway, I'll Look around other ways to fix it.

**Comment by jeffposnick** [comment]
I don't think this is an issue anymore with the latest `create-react-app` setup. See https://github.com/jeffposnick/create-react-pwa/compare/c-r-a-0.6.0...c-r-pwa-0.6.0
